tower-http/src/decompression/mod.rs

```diff
@@ -53,15 +97,22 @@
 //! # }
 //! ```
 
+mod request;
+
 mod body;
 mod future;
 mod layer;
 mod service;
 
-pub use self::{
-    body::DecompressionBody, future::ResponseFuture, layer::DecompressionLayer,
-    service::Decompression,
-};
+pub use self::body::DecompressionBody;
+
+pub use self::future::ResponseFuture;
+pub use self::layer::DecompressionLayer;
+pub use self::service::Decompression;
+
+pub use self::request::future::RequestDecompressionFuture;
+pub use self::request::layer::RequestDecompressionLayer;
+pub use self::request::service::RequestDecompression;
```

Comment:
Why did you change these imports? Wanna revert the change? It makes things easier to review.

Reply:
I wanted to separate the import of the `DecompressionBody` from the response imports. I will revert the change.